stacks/kafka-flink/test.sh

```diff
@@ -97,7 +96,7 @@ function feed-data() {
   echo
 
   # Wait a bit for the data to transfer and converge successfully.
-  sleep 1
+  sleep 2
```

Comment:
you can make it 5 to be on the safe side.

Reply:
I've increased to to three seconds. Unless we get errors, I think it is fine.